client/rest/src/main/java/org/opensearch/client/RestClient.java

```diff
@@ -149,6 +173,7 @@ public class RestClient implements Closeable {
         this.nodeSelector = nodeSelector;
         this.warningsHandler = strictDeprecationMode ? WarningsHandler.STRICT : WarningsHandler.PERMISSIVE;
         this.compressionEnabled = compressionEnabled;
+        this.chunkedEnabled = null;
```

Comment:
`Optional.empty()` probably ... (or we could go with `Boolean`)

Reply:
Updated. Removed null checks and using `Optional.empty` everywhere.